### PR TITLE
Delete unused code & Add webpack plugin to inspect unused js files

### PR DESCRIPTION
**Pull request by kaviththiranga**

### CODE REVIEW

File: modules/web/webpack.config.js [hunk 1]
```diff
@@ -85,7 +86,8 @@ var config = {
     },
     plugins: [
         extractCSSBundle,
-        extractThemes
+        extractThemes,
+        new UnusedFilesWebpackPlugin({pattern: '**/*.js'})
```
Comment (roonyh) [inline modules/web/webpack.config.js]: Can we capture `.jsx` files as well? how about `**/*.*`?

Comment (kaviththiranga) [inline modules/web/webpack.config.js]: *.* will complain about unused images and css too right? can we guarantee there is no css or svg which is not included using webpack aware api

Comment (kaviththiranga) [inline modules/web/webpack.config.js]: WDYT?

Comment (kaviththiranga) [inline modules/web/webpack.config.js]: or may be we can set it as `js/**/*.*`

Comment (roonyh) [inline modules/web/webpack.config.js]: Ah. yeah. `'**/*.js*'` should capture jsx as well ne?

**Comment by kaviththiranga** [comment]
@roonyh Updated. Please check

**Comment by roonyh** [comment]
LGTM :+1:

**Pull request merged**